rlclientlib/azure_factories.cc

```diff
@@ -31,13 +31,35 @@ namespace reinforcement_learning {
   }
 
   int restapi_data_transport_create(m::i_data_transport** retval, const u::configuration& config, i_trace* trace_logger, api_status* status) {
-    const auto uri = config.get(name::MODEL_BLOB_URI, nullptr);
-    if (uri == nullptr) {
-      RETURN_ERROR(trace_logger, status, http_uri_not_provided);
-    }
-    i_http_client* client;
-    RETURN_IF_FAIL(create_http_client(uri, config, &client, status));
-    *retval = new m::restapi_data_transport(client, trace_logger);
+      auto uri = config.get(name::MODEL_BLOB_URI, nullptr);
```

Comment:
Line 35 to line 62: the indent seems not aligned.

Reply:
indent done through formatdocument in visualstudio